### PR TITLE
optimize bullet and fix js sync to native

### PR DESCRIPTION
**Pull request by JayceLai**
Re: cocos-creator/3d-tasks#

Changelog:
 *

### CODE REVIEW

**Comment by JayceLai** [comment]
https://github.com/cocos-creator/3d-tasks/issues/7487
https://github.com/cocos-creator/3d-tasks/issues/7344

File: cocos/profiler/profiler.ts [hunk 1]
```diff
@@ -40,6 +40,8 @@ import { preTransforms } from '../core/math/mat4';
 
 const _characters = '0123456789. ';
 
+const _average = 1e3;
```
Comment (pandamicro) [inline cocos/profiler/profiler.ts]: 1s is too long, please keep 500ms

File: cocos/profiler/profiler.ts [hunk 2]
```diff
@@ -173,6 +176,7 @@ export class Profiler {
             this._showFPS = true;
             this._canvasDone = true;
             this._statsDone = true;
+            legacyCC.game.config.showFPS = true;
```
Comment (pandamicro) [inline cocos/profiler/profiler.ts:179]: What's the purpose of this ?

Comment (JayceLai) [reply cocos/profiler/profiler.ts:179]: User may call 'cc.profiler.showStats' or 'cc.profiler.hideStats' to change showing state, after the call, redisplaying the screen will cause the debug panel to return to its previous state if not sync the configuration modification.

This is actually just to sync the configuration modification in debug.setDisplayStats 
image

**Review by SantyWang — APPROVED** [review]

**Review by pandamicro — APPROVED** [review]

**Pull request merged**